lib/index.js

```diff
@@ -70,6 +70,13 @@ var Horseman = function(options) {
 		phantomOptions['cookies-file'] = this.options.cookiesFile;
 	}
 
+	if (this.options.debugPort) {
+		phantomOptions['remote-debugger-port'] = this.options.debugPort;
+		if (this.options.debugAutorun !== false) {
+			phantomOptions['remote-debugger-autorun'] = 'yes';
+		}
```

Comment:
You should probably have an `else` here that actually sets the option to `'no'`, rather than relying on it defaulting to `'no'`.